Dockerfile

```diff
@@ -14,10 +14,37 @@ RUN apt-get update -y && \
 
 RUN apt-get update -y && \
   apt-get upgrade -y && \
-  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig php7.0 php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu && \
+  apt-get install -y git-core unzip npm nodejs-legacy wget fontconfig libaio1 php7.0 php7.0-xml php7.0-mbstring php7.0-curl php7.0-gd php7.0-zip php7.0-intl php7.0-sqlite3 php7.0-mysql php7.0-pgsql php7.0-soap php7.0-phpdbg php-redis php-memcached php-imagick php-smbclient php-apcu php7.0-dev && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+#ORACLE INSTALLATION
+#get the zips & unzip them & set paths & install oci8
+RUN mkdir /opt/oracle && \
+  wget -O /opt/oracle/instantclient-basic-linux.x64-12.1.0.2.0.zip \
```

Comment:
Maybe we should add these files statically to the repo? We already had failed builds because of failing downloads.